docs/grpc/c#.md

```diff
@@ -63,10 +72,20 @@ System.Environment.SetEnvironmentVariable("GRPC_SSL_CIPHER_SUITES", "HIGH+ECDSA"
             
 // Lnd cert is at AppData/Local/Lnd/tls.cert on Windows
 // ~/.lnd/tls.cert on Linux and ~/Library/Application Support/Lnd/tls.cert on Mac
-var cert = File.ReadAllText(<Tls_Cert_Location>);
-
-var sslCreds = new SslCredentials(cert);
-var channel = new Grpc.Core.Channel("localhost:10009", sslCreds);
+var rawCert = File.ReadAllBytes(<Tls_Cert_Location>);
+var cert = new X509Certificate2(rawCert);
+var httpClientHandler = new HttpClientHandler
+{
+    // HttpClientHandler will validate certificate chain trust by default. This won't work for a self-signed cert. So turn it off as a hack.
+    ServerCertificateCustomValidationCallback = (httpRequestMessage, cert, cetChain, policyErrors) => true
+};
+httpClientHandler.ClientCertificates.Add(cert);
```

Comment:
The TLS cert that's read above (`rawCert`) isn't a _client_ certificate. It does not authenticate the client against the server but is used by the client to make sure it's talking to the correct server.
Therefore you shouldn't use the `ServerCertificateCustomValidationCallback` hack but instead use the `rawCert` as the trusted certificate.

If we're updating the docs anyway, can you also mention in this section that when an IP address is used instead of `localhost` that the address needs to be added to `lnd`'s configuration? See https://github.com/lightningnetwork/lnd/issues/5764.

Reply:
That's a great observation. I thought the cert had to be passed as a client cert, because it was used as `SslCredentials` in the old example.

- Fixed that server cert check
- Removed the client cert thing
- Added a note about the IP address to the docs of all languages

Reply:
Thanks, looks good! Can you please squash the 3 commits, use a prefixed commit message as described in the contribution guidelines and add an entry to the release notes?

Reply:
Squashed, prefixed and added to release notes (I suppose 0.14 is the right release?)

Reply:
Just added myself to the contributors in the release notes just now, for the personal fame ;-)